### DISCUSSION TITLE
Polyglot in Dungeon World?

### DISCUSSION THREAD

**Discussion by FederSchakal** [mclemente/fvtt-module-polyglot#233] [Q&A] [answered]
Hey,
how can i use Polyglot in Dungeon World?
I can select the languages, but my test player cannot read them. In which item should I put them? Or is it not possible in Dungeon World?

**Comment by mclemente** [comment, marked as answer]
Add any item (move, equipment, anything), as long as it's labeled like `Language (Name)`, it should work. Example: if you have Elvish and Dwarvish languages, add moves labeled `Language (Elvish)` and `Language (Dwarvish)`.

**Comment by FederSchakal** [comment]
Ok thanks it worked!

Does the label also work in German "Sprache (Name)" or only in English?

**Reply by mclemente** [reply]
Yes, it does. Actually, if your system is set to German, it will only work with Sprache.

**Reply by FederSchakal** [reply]
System is in German, unfortunately it only works with Language (Name).

Thanks anyway ^^
